kie-internal/src/main/java/org/kie/internal/builder/fluent/DMNRuntimeFluent.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.internal.builder.fluent;
+
+public interface DMNRuntimeFluent
+        extends DMNFluent<DMNRuntimeFluent, ExecutableBuilder>,
+                ContextFluent<DMNRuntimeFluent, ExecutableBuilder> {
+
+    static DMNRuntimeFluent create(CommandBasedExecutable executable) {
+        try {
+            return (DMNRuntimeFluent) Class.forName("org.kie.dmn.core.fluent.DMNRuntimeFluentImpl")
```

Comment:
I believe using `Class.forName` is less preferable than using `org.kie.internal.utils.ClassLoaderUtil`.
I understand this is -api and not -internal but hence might need a bit of refactor here to reconciliate the different requirements?

Reply:
Updated both `DMNRuntimeFluent` and `ExecutableBuilder`. Btw I still think we should remove all this stuff an move to a different approach soon :smile: